libs/util/code-gen-types/src/code-gen-types.ts

```diff
@@ -225,6 +225,16 @@ export type PropertyTypeDef = Omit<models.PropertyTypeDef, "type"> & {
 
 export type PluginInstallation = BlockOmittedFields<models.PluginInstallation>;
 
+export type Package = BlockOmittedFields<models.Package>;
+
+export type PackageFile = {
+  package: Package;
+  path: string;
+  originalFileChecksum: string;
+  fullContent: string;
+  diff: string;
+};
```

Comment:
why do we need this? (asking for knowledge)

Reply:
@yuval-hazaz For reusing this type in the future (maybe)